scripts/brownie-tests/contracts/Sweeper.vy

```diff
@@ -0,0 +1,43 @@
+# @version 0.3.7
```

Comment:
Let's not include our contracts here. These are currently in a private repo and I don't think they are ready to release.

Reply:
oops, my bad! should i try to remove them from the commit, or is it fine to just not merge them into main (technically this one is public now..)